docs/cookbook/tasks/task_dependencies.rst

```diff
@@ -65,10 +65,16 @@ Create A Dependency
 
 Tasks each have an ``upstream_tasks`` field and a ``downstream_tasks`` field. Each field is a 
 list ``[]`` type and can contain zero, one, or multiple Task entity dictionaries representing the 
-dependent Tasks. Here is how to create a dependency between our "Layout" and "Anim" Tasks::
+dependent Tasks.
+There are four dependency types from which you can choose: ``finish-to-start-next-day``, ``start-to-finish-next-day``, ``start-to-start``, ``finish-to-finish``.
+If no dependency type provided the default ``finish-to-start-next-day`` would be used. 
```

Comment:
If no dependency type **is** provided, the default ``finish-to-start-next-day`` **will** be used.